doc/api/child_process.md

```diff
@@ -147,7 +147,9 @@ added: v0.1.90
 * Returns: {ChildProcess}
 
 Spawns a shell then executes the `command` within that shell, buffering any
-generated output.
+generated output. When specifying paths, spaces need to be dealt with 
```

Comment:
That's a good point! For some reason I was just focused on the path aspect :man_shrugging: 

What about something like: 
...The `command` string passed to the exec function is processed directly by the shell and special characters need to be dealt with 
accordingly, eg. `exec("'/path/to/test file/test.sh' arg1 arg2")` or 
`exec("echo \"Hello \$\\ World\"")`.